bundles/org.openhab.binding.wemo/src/main/resources/OH-INF/config/insight.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:wemo:insight">
+		<parameter name="udn" type="text" required="true">
+			<label>Unique Device Name</label>
+			<description>The UDN identifies the WeMo Insight Switch</description>
+		</parameter>
+		<parameter name="currentPowerSlidingSeconds" type="integer" min="0" unit="s">
+			<label>Current Power sliding window</label>
+			<description>Sliding window in seconds for which moving average power is calculated (0 = disabled)</description>
+			<unitLabel>seconds</unitLabel>
```

Comment:
Just for my information: what is this tag compared to "unit" attribute ?

Reply:
Please see https://www.openhab.org/docs/developer/bindings/config-xml.html#xml-structure-for-configuration-descriptions.

Reply:
Makes no sense IMHO to hardcode an English unit label. The label should be automatically derived from the unit attribute.

Reply:
Is this label considered by MainUI ?
Is it translated (i18n) through an entry in the i18n properties ?

Reply:
I found this tag used 86 times.

Reply:
It might be obsoleted, in which case we need to update the documentation and remove it from the codebase. I didn't check it yet.